### DISCUSSION TITLE
How do I delete a forked GitHub repo that's unavailable due to DCMA takedown?

### DISCUSSION THREAD

**Discussion by wqdygkd** [community/community#28484] [Other Feature Feedback, Questions, & Ideas]
Does anyone know how to delete a forked GitHub repo that's been subject to a DCMA takedown? The repo is there (with a DCMA banner) but I can't do anything to it. I've tried clicking the 'settings' tab but nothing happens

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there, but support should be able to. Good luck! :slightly_smiling_face:

**Comment by double12gzh** [comment]
> Does anyone know how to delete a forked GitHub repo that's been subject to a DCMA takedown? The repo is there (with a DCMA banner) but I can't do anything to it. I've tried clicking the 'settings' tab but nothing happens

Hi, I am writing to confirm whether you've already deleted it successfully? Could pls share with me how did you do that ? Thanks.
